app/src/main/java/com/infomaniak/mail/workers/DraftsActionsWorker.kt

```diff
@@ -342,15 +374,38 @@ class DraftsActionsWorker @AssistedInject constructor(
             }
         }
 
+        suspend fun executeScheduleAction() = with(ApiRepository.scheduleDraft(mailboxUuid, draft, okHttpClient)) {
+            when {
+                isSuccess() -> {
+                    scheduleAction = data?.scheduleAction
+                    realmActionOnDraft = deleteDraftCallback(draft)
+                    refreshScheduledDraftsFolder()
```

Comment:
Doesn't look right that we need to refresh a folder after executing a draft action.
It's supposed to already be handled somewhere else.

Reply:
Fixed.